ingestors/documents/pdf.py

```diff
@@ -20,8 +28,13 @@ def ingest(self, file_path, entity):
         """Ingestor implementation."""
         try:
             self.parse_and_ingest(file_path, entity, self.manager)
+        except pikepdf._qpdf.PasswordError as pwe:
+            raise ProcessingException(
+                "Could not extract PDF file. The PDF is protected with a password. Try removing the password protection and re-ingesting the documents."
+            )
         except Exception as ex:
-            raise ProcessingException("Could not extract PDF file: %r" % ex) from ex
+            log.info(f"Failed to ingest PDF: {file_path}. Reason: {ex}.")
```

Comment:
You could use the [exception](https://docs.python.org/3/library/logging.html#logging.Logger.exception) method on the logger, which will print the stack trace as well.